packages/react-components/react-color-picker-preview/library/etc/react-color-picker-preview.api.md

```diff
@@ -76,12 +76,11 @@ export const colorSliderClassNames: SlotClassNames<ColorSliderSlots>;
 // @public
 export type ColorSliderProps = Omit<ComponentProps<Partial<ColorSliderSlots>, 'input'>, 'defaultValue' | 'onChange' | 'value'> & {
     channel?: string;
-    defaultValue?: number;
     max?: number;
     min?: number;
```

Comment:
Shouldn't we remove min/max from props as well?

Reply:
removed